src/containers/About/About.js

```diff
@@ -154,7 +175,34 @@ export /* istanbul ignore next */ class About extends Component {
             defaultMessage: 'About'
           })}
         </h1>
-        <Table headers={headers} rows={rows} loading={loading} />
+        <div data-testid="dashboard-table">
+          <Table
+            title="Dashboard"
+            headers={headers}
+            rows={dashboardRows}
+            loading={loading}
+          />
+        </div>
+        <div data-testid="pipelines-table">
+          <Table
+            title="Pipelines"
+            headers={headers}
+            rows={pipelineRows}
+            loading={loading}
+            emptyTextAllNamespaces="Pipelines not installed"
+            emptyTextSelectedNamespace="Pipelines not installed"
+          />
+        </div>
+        <div data-testid="triggers-table">
```

Comment:
since Triggers is optional can we hide this table if it's not installed?

Reply:
done